README.rst

```diff
@@ -24,7 +24,7 @@ Community support is available at `Galaxy Help <https://help.galaxyproject.org/>
 Galaxy Quickstart
 =================
 
-Galaxy requires Python 3.8 . To check your Python version, run:
+Galaxy requires Python 3.8 or higher. To check your Python version, run:
```

Comment:
https://github.com/galaxyproject/galaxy/issues/18849#issuecomment-2360564119 not sure if this should include the upper bound

Reply:
I'd rather put an upper bound on the release notes for each version than here, we strive to support the latest Python version on dev.